src/modules/api/solver-registration/services/tests/solver-registration.service.spec.ts

```
@@ -0,0 +1,363 @@
+    // Now replace it with our mock
+    (service as any).apiRequestExecutor = mockApiRequestExecutor;
+  });
+
+  beforeEach(() => {
+    // Reset mockConfig to default values before each test
+    mockConfig.registrationEnabled = true;
+    mockConfig.registrationBaseUrl = 'https://api.example.com';
+    mockConfig.apiUrl = 'https://api.example.com/api/v1/solverRegistry/registerSolver';
+    mockConfig.registrationPrivateKey =
+      '0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef';
+  });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)